src/app/orders/container.rs

```diff
@@ -1,37 +1,31 @@
 use crate::app::orders::{proxy::OrdersProxy, Orders};
 use crate::app::{
-    App, CmdHandle, CmdManager, Effect, Notification, RenderInfo, ShouldRender, StreamHandle,
-    StreamManager, SubHandle, UndefinedGMsg,
+    effect::Effect, App, CmdHandle, Notification, RenderInfo, ShouldRender, StreamHandle, SubHandle,
```

Comment:
I think I've removed this `effect::` in recent commits. Git / GitHub wasn't probably clever enough to show it to you as a conflict.